meta-lmp-bsp/recipes-bsp/u-boot/u-boot-base-scr/uz3eg-iocc/uEnv.txt.in

```diff
@@ -0,0 +1,11 @@
+fdt_file=system-top.dtb
+fpga_image=bitstream.bit.bin
+optee_ovl_addr=0x22000000
+fpga_addr=0x18000000
+bootcmd_fpga=fatload ${devtype} ${devnum}:1 ${fpga_addr} ${fpga_image}; fpga load 0 ${fpga_addr} ${filesize}
+bootcmd_tee_ovy=fdt addr ${fdt_addr_r}; fdt resize 0x1000; fdt apply ${optee_ovl_addr}
+bootcmd_dtb=fatload ${devtype} ${devnum}:1 ${fdt_addr_r} ${fdt_file}
+bootcmd_load_k=fatload ${devtype} ${devnum}:1 ${kernel_addr_r} Image
+bootcmd_args=setenv bootargs console=ttyPS0,115200 root=/dev/mmcblk1p2 rootwait rw
+bootcmd_run=booti ${kernel_addr_r} - ${fdt_addr_r}
+bootcmd=run bootcmd_dtb; run bootcmd_tee_ovy; run bootcmd_fpga; run bootcmd_args; run bootcmd_load_k; run bootcmd_run
```

Comment:
why is this a .in file since there is nothing in here to be substituted

Reply:
This is just because the recipe expects a file named like that, in case there is something to be dynamically replaced (which is not required for this file in particular).

.in would be used in cases such as rpi, which would have a different boot command for 32 and 64 bits.